Source/Core/UpdaterCommon/UpdaterCommon.cpp

```diff
@@ -695,31 +693,32 @@ bool RunUpdater(std::vector<std::string> args)
   UI::SetVisible(false);
 
   Common::ScopeGuard ui_guard{[] { UI::Stop(); }};
-  Options opts = std::move(*maybe_opts);
+  auto [this_manifest_url, next_manifest_url, content_store_url, install_base_path,
+        binary_to_restart, parent_pid, log_file_] = std::move(*maybe_opts);
```

Comment:
Would something like `log_file_path` or `log_file_name` be more appropriate here? Using the trailing underscore for disambiguation is fine, but this one feels like it is a different type (and purpose) from `log_file` altogether.